client/src/locales/en-US.json

```diff
@@ -583,6 +584,13 @@
             "browse": "Go to location",
             "failedDetails": "Please make sure you are online."
         },
+        "RestoreFile": {
+            "restore": "Restoration of {name}",
```

Comment:
It felt a little weird because this is displayed in the "Last uploaded" section and there is no "Import of {file}" for the other actions: 
![image](https://github.com/user-attachments/assets/32c0c10e-87ac-4ee5-8f66-c627f6efd2d2)

Since there is a specific icon for restore, maybe it is best to remove the "Restoration of"?

Or, if you prefer, I can open a new issue to discuss how we can harmonize this dialog:
- Add a "restored" section?
- Rename "Last uploaded" in "Done" or "Completed"?
- There multiple words for the same thing : "background _tasks_", "last _uploaded_", _import_...